includes/Widgets/Stories.php

```diff
@@ -0,0 +1,393 @@
+<?php
+/**
+ * Stories Widgets.
+ *
+ * @package Google\Web_Stories
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Widgets;
+
+use Google\Web_Stories\Story_Query;
+use WP_Widget;
+use function Google\Web_Stories\get_layouts;
+
+/**
+ * Class Stories
+ *
+ * @package Google\Web_Stories\Widgets
+ */
+class Stories extends WP_Widget {
+
+	/**
+	 * Widget args.
+	 *
+	 * @var array
+	 */
+	public $args = [
+		'before_title'  => '<h4 class="widgettitle web-stories-widget-title">',
+		'after_title'   => '</h4>',
+		'before_widget' => '<div class="widget-wrap web-stories-widget-wrapper">',
+		'after_widget'  => '</div>',
+	];
+
+	/**
+	 * Stories constructor.
+	 *
+	 * @return void
+	 */
+	public function __construct() {
+		$id_base        = 'web_stories_widget';
+		$name           = __( 'Web Stories', 'web-stories' );
+		$widget_options = [
+			'description' => __( 'Display Web Stories in Sidebar Section.', 'web-stories' ),
+			'classname'   => 'google-stories-widget',
+		];
+
+		parent::__construct( $id_base, $name, $widget_options );
+	}
+
+	/**
+	 * Output widget.
+	 *
+	 * phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+	 *
+	 * @param array $args Widget args.
+	 * @param array $instance Widget instance.
+	 *
+	 * @return void
+	 */
+	public function widget( $args, $instance ) {
+		echo $instance['before_widget'];
+
+		if ( ! empty( $instance['title'] ) ) {
+			echo $args['before_title'] . apply_filters( 'widget_title', $instance['title'] ) . $args['after_title'];
+		}
+
+		$story_attrs = [
+			'view_type'                 => $instance['view-type'],
+			'show_title'                => (bool) $instance['show_title'],
+			'show_author'               => (bool) $instance['show_author'],
+			'show_date'                 => (bool) $instance['show_date'],
+			'show_excerpt'              => (bool) $instance['show_excerpt'],
+			'list_view_image_alignment' => ( (bool) $instance['image_align_right'] ) ? 'right' : 'left',
+			'show_stories_archive_link' => (bool) $instance['archive_link'],
+		];
+
+		$story_args = [
+			'posts_per_page' => $instance['number'],
+		];
+
+		echo ( new Story_Query( $story_attrs, $story_args ) )->render();
```

Comment:
```suggestion
		$story_query = new Story_Query( $story_attrs, $story_args );
		echo $story_query->render();
```
Nic pic, but this is much easier to read.